presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlIntegrationMixedCaseTest.java

```diff
@@ -187,11 +187,11 @@ public void testDuplicatedRowCreateTable()
                 "line 1:49: Column name 'orderkey' specified more than once");
 
         assertQueryFails("CREATE TABLE test (a integer, A integer)",
-                "line 1:31: Column name 'A' specified more than once");
+                "Duplicate column name 'A'");
```

Comment:
Hmm, are we losing the line/column information from the error? I feel that is useful for large queries.

Reply:
So I remember it right, this is coming from MySQL Exception, which is why I think this was modified - 
```
SQL Error [1060] [42S21]: Duplicate column name 'A'
```